FIL/sub-noise_ses-001_task-noise220622_run-001_positions.tsv

```diff
@@ -0,0 +1,69 @@
+name	Px	Py	Pz	Ox	Oy	Oz
```

Comment:
are these column names part of the standard?

Reply:
No, this is a "manufacturer specific" file. We took a guess on this before BIDS-MEG was finalised. I'll update the README to reflect this.